cli/cmd/lql.go

```diff
@@ -162,6 +177,12 @@ func init() {
 		"validate_only", "", false,
 		"validate query only (do not run)",
 	)
+	// fail on count
+	queryRunCmd.Flags().StringVarP(
+		&queryCmdState.FailOnCount,
+		"fail_on_count", "", "",
+		"fail if the query matches the fail_on_count expression",
```

Comment:
nit: normally we don't reference a flag name that is not a different one, I would suggest this but I am biased so, let's ask @dmurray-lacework for his opinion too
```suggestion
		"fail if the results from a query match the provided expression",
```

Reply:
I agree with the suggested change, if the flag name isn't apparent then the description should be worded differently

Reply:
Fixed.